packages/webpack/src/runtime/runtime.js

```diff
@@ -0,0 +1,189 @@
+// @ts-check
+/// <reference path="./runtime.d.ts" />
+/* global LAVAMOAT */
+/* global lockdown, harden, Compartment */
+
+const {
+  create,
+  freeze,
+  assign,
+  defineProperty,
+  defineProperties,
+  getOwnPropertyDescriptors,
+} = Object
+const warn = typeof console === 'object' ? console.warn : () => {}
+// Avoid running any wrapped code or using compartment if lockdown was not called.
+// This is for when the bundle ends up running despite SES being missing.
+// It was previously useful for sub-compilations running an incomplete bundle as part of the build, but currently that is being skipped. We might go back to it for the sake of build time security if it's deemed worthwihile in absence of lockdown.
```

Comment:
I am unclear on the tradeoff here vs just aborting instead.  What does "switching to no-op" mean?

Reply:
discussed in person